### PR TITLE
Drop support for Python 2.6 and 3.2.

### PR DESCRIPTION
**Pull request by tseaver**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26,py27,py32,py33,py34,pypy,pypy3,coverage,docs
+envlist = py27,py33,py34,pypy,pypy3,coverage,docs
```
Comment (jamadden) [inline tox.ini]: There's a [testenv:py26] block that could go.

It adds a dependency on ordereddict, but that doesn't seem to be from this package. So probably somewhere else there's going to be some code catching ImportError that can go...

**Comment by jamadden** [comment]
Aww, I just *added* support for Pyton 3.2 :)

Other than my one comment, LGTM, pending Travis.

File: tox.ini (hunk 1 above)
Comment (tseaver) [inline tox.ini]: b1c86c5  drops that block.  I can't find any actual use of `ordereddict` anywhere in the source.

**Comment by tseaver** [comment]
3.2 can't be tested on Travis any longer.

File: tox.ini (hunk 1 above)
Comment (jamadden) [inline tox.ini]: I can't either. I'm guessing it was transitive from some other package.

**Comment by jamadden** [comment]
I know, and I'm not upset, I was just having some fun, because I remember that while initially adding a bunch of 3.2 support I wound up breaking the build of a lot of other zope downstream packages. After which I quit trying :)

**Pull request merged**